cmp/grid/columns/Column.ts

```diff
@@ -606,8 +606,7 @@ export class Column {
         this.rendererIsComplex = rendererIsComplex;
         this.highlightOnChange = highlightOnChange;
 
-        this.tooltip = tooltip;
-        this.tooltipElement = tooltipElement;
+        this.tooltip = tooltip ?? tooltipElement;
```

Comment:
If we are just deprecating and not removing, should this be tooltipElement ?? tooltip since the former is what currently takes precedence?

Reply:
It seemed wrong to me to prioritise a deprecated property over the correct one, even if that meant changing the current behaviour slightly.

I could go either way. You're not expected to provide both (we had a warning before), so I don't think the order matters too much.